src/app/workflow/versions/versions.component.css

```diff
@@ -39,5 +39,10 @@ th {
 /* Highlight row with the selected version */
 /* The mat-row is so that this is more specific and able to override the hover style above */
 tr.mat-row.highlight {
-  background-color: rgba(0, 0, 0, 0.12);
+  background-color: rgba(232, 234, 246);
```

Comment:
This colour is #e8eaf6 which is in our material colour scheme so you could use the color from the palette. Not sure if you can use it for the `.mat-row:hover` though because you're specifying the transparency. If you can't then I would just add a comment saying that the colour is from the colour scheme.

Reply:
mm thanks for pointing that out, I made that change before I knew about the colour palette. I'm updating that now, I'll also be converting the file to scss.  The transparency is also doable so that'll be changed as well